book/_toc.yml

```diff
@@ -46,19 +46,12 @@ parts:
             - file: tutorials/database/5_plot_raster_example
             - file: tutorials/database/6_exporting_data
             - file: tutorials/database/7_wrap_up
-        - file: tutorials/OOP/index
-          title: Object Oriented Programming
-          sections:
-            - file: tutorials/OOP/OOP_tutorial
-            - file: tutorials/OOP/alien
-        - file: tutorials/lidar/index
-          title: Lidar Remote Sensing of Snow
-          sections:
-            - file: tutorials/lidar/1_introduction_to_lidar
-            - file: tutorials/lidar/2_elevation_differencing
-            - file: tutorials/lidar/3_common_pitfalls
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
need to remove all the github markup from past merge conflicts

Reply:
done, thanks!